src/Core/BoundedContexts/Interviewer/WB.Core.BoundedContexts.Interviewer/Views/Dashboard/DashboardViewModel.cs

```diff
@@ -538,6 +553,7 @@ public void ChangeWorkspace(string workspaceName)
             memoryCacheSource.ClearAll();
 
             ViewModelNavigationService.NavigateToDashboardAsync();

```

Comment:
add ```await```

Reply:
Added but would not make any difference since it's a click listener for void call. It would not be awaited anyway.